README.md

```diff
@@ -39,9 +38,46 @@ The package could be installed with composer:
 composer require yiisoft/cache-db --prefer-dist
 ```
 
+## Create Table
+
+The package provides a `DbHelper::class` that creates the cache table for default `{{%cache}}`. You can use it as follows:
+
+```php
+DbHelper::ensureTable($db);
+```
+
+For custom table name you can use:
+
+```php
+DbHelper::ensureTable($db, '{{%custom_cache_table}}');
+```
+
+> Note: Additionally you can import the `RAW SQL` directly to create the tables.
+>
+>- [schema-mssql](/docs/en/migration/schema-mssql.sql).
+>- [schema-mysql](/docs/en/migration/schema-mysql.sql).
+>- [schema-oracle](/docs/en/migration/schema-oci.sql).
+>- [schema-pgsql](/docs/en/migration/schema-pgsql.sql).
+>- [schema-sqlite](/docs/en/migration/schema-sqlite.sql).
+
+## Drop Table
+
+For dropping table you can use:
+
+```php
+DbHelper::dropTable($db);
+```
+
+For custom table name you can use:
+
+```php
+DbHelper::dropTable($db, '{{%custom_cache_table}}');
+```
+
```

Comment:
```suggestion
### Database Preparing

Package provides two way for preparing database:

1. Raw SQL. You can use it with the migration package used in your application.

- [MSSQL](/docs/en/migration/schema-mssql.sql),
- [MySQL / MariaDB](/docs/en/migration/schema-mysql.sql),
- [Oracle](/docs/en/migration/schema-oci.sql),
- [PostgreSQL](/docs/en/migration/schema-pgsql.sql),
- [SQLite](/docs/en/migration/schema-sqlite.sql),

2. `DbHelper` for create/drop cache table (by default `{{%cache}}`).

```php
// Create table with default name
DbHelper::ensureTable($db);

// Create table with custom name
DbHelper::ensureTable($db, '{{%custom_cache_table}}');

// Drop table with default name
DbHelper::dropTable($db);

// Drop table with custom name
DbHelper::dropTable($db, '{{%custom_cache_table}}');
```

> Note: don't use `DbHelper` in migrations, because migrations should not have external dependencies.